homeassistant/components/marytts/tts.py

```diff
@@ -86,4 +86,4 @@ async def async_get_tts_audio(self, message, language, options=None):
 
         data = self._mary.speak(message, effects)
 
-        return self._mary.codec, data
+        return self._mary.codec.replace("_FILE", "").replace("WAVE", "WAV"), data
```

Comment:
We use normaly simple maps for that:
```
MAP_MARYTTS_HA = {
    "WAV_FILE": "wav"
}
```

Reply:
Thanks for your feedback! I have now used an approach using mapping.